pkg/complianceoperator/gvk.go

```diff
@@ -0,0 +1,28 @@
+package complianceoperator
+
+import (
+	compv1alpha1 "github.com/ComplianceAsCode/compliance-operator/pkg/apis/compliance/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	requiredObjectTypes = []runtime.Object{
```

Comment:
nit:  I eventually reconciled this on my own, but a comment here that this relates to the CRDs might be helpful as I was initially and incorrectly thinking this referred to the CRs.